src/cdk/stepper/stepper.ts

```diff
@@ -131,6 +152,10 @@ export class CdkStepper {
     this._groupId = nextId++;
   }
 
+  ngAfterContentChecked(): void {
```

Comment:
This is called every change detection run, which seems like it may be overkill. Can you clarify why this is necessary? Would `ngAfterContentInit` work here?